Package.swift

```diff
@@ -14,6 +14,7 @@ import PackageDescription
 
 var package = Package(
```

Comment:
@natecook1000 can we take this opportunity to make the indentation 2 spaces?

Reply:
Oh sure. It should be 4 in code samples and example code and 2 everywhere else.